### DISCUSSION TITLE
Use images from RaSCSI?

### DISCUSSION THREAD

**Discussion by score100** [ZuluSCSI/ZuluSCSI-firmware#54] [Q&A] [answered]
Hi there,
I'm thinking to step over from rascsi to zulu. Could I use my existing .hda images of hard drives used in Rascsi for Zulu? They're all formatted with and used for an E-MU ESI-32.

**Comment by aperezbios** [comment, marked as answer]
Yes, they should work fine with no modification. "HDA" is not its in any way own unique format; It's just a raw HDD image. For nearly all Macintosh users, this means an Apple Partition Map-formatted disk image, with at least one HFS or HFS+ filesystem on it. 

Does that help?

**Reply by score100** [reply]
Perfect, just what I wanted to check :)
